code/workspaces/infrastructure-api/resources/jupyterlab.headless-service.template.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Given the two service templates are identical I wonder if we could just have one? Especially if it is renamed as a spark service